### ISSUE TITLE
data-f-bind and templating: templated element doesn't redraw

### ISSUE THREAD

**Issue by mmrj** [forio/flow.js#27]
env: using https://forio.com/tools/js-libs/1.4.2/epicenter.js and https://forio.com/tools/js-libs/flow/0.8.3/flow.js (from 5/18 rebuild) and lodash.

the following code works great the first time i load the page:
```
            <td data-f-bind="sampleInt, sampleString">
                The data-f-bind is in the "td" tag, we'll use templates to show the values of sampleInt <%= sampleInt %> and sampleString <%= sampleString %>.
            </td>
```

however, if i have sampleInt and/or sampleString also bound (data-f-bind) in other elements on the page, and use those other elements to change the value, then the entire table cell above becomes empty. it's like the templating is causing the td to not be redrawn.

**Comment by narenranjit** [comment]
I think this is independent of other things on the page. Bind with templates works fine "on load" when the page initializes the first time, but doesn't work for subsequent updates.

**Comment by mmrj** [comment]
makes sense. it was only obvious to me when i was otherwise making changes to those variables, because that's all that my test page does.

**Comment by mmrj** [comment]
fwiw, the templating DOES update/redraw with `data-f-foreach`. for example:

```
            <td>here is a list of the fields in my sales manager object:
                <ul data-f-foreach="salesMgr">
                    <li>Field is <%= key %> and value is <%= value %></li>
                </ul>
            </td>
```

if elsewhere on the page (in a different data-f-bind element) i update the value of one of the elements of the salesMgr object, this table cell is redrawn and the updated value is displayed.

**Comment by narenranjit** [comment]
Fixed in -edge, will be in upcoming 0.9

**Comment by mmrj** [comment]
verified in 0.9.0
